packages/mesh/src/Mesh.js

```diff
@@ -85,6 +87,31 @@ export default class Mesh extends RawMesh
         this.tint = 0xFFFFFF;
 
         this.blendMode = BLEND_MODES.NORMAL;
+
+        /**
+         * TextureMatrix instance for this Mesh, used to track Texture changes
+         *
+         * @member {PIXI.TextureMatrix}
+         * @private
+         */
+        this.uvMatrix = new TextureMatrix(this._texture);
```

Comment:
Make sure to destroy uvMatrix (and remove texture reference) on destroying Mesh

Reply:
There are no destoy method in mesh since v4. Lets fix it in different PR